streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/SmokeTestDriverIntegrationTest.java

```diff
@@ -99,8 +115,17 @@ SmokeTestDriver.VerificationResult result() {
     // We set 2 timeout condition to fail the test before passing the verification:
     // (1) 10 min timeout, (2) 30 tries of polling without getting any data
     @ParameterizedTest
-    @CsvSource({"false, false", "true, false"})
-    public void shouldWorkWithRebalance(final boolean stateUpdaterEnabled, final boolean processingThreadsEnabled) throws InterruptedException {
+    @CsvSource({
+        "false, false, true",
+        "true, false, true",
+        "false, false, false",
+        "true, false, false",
```

Comment:
I removed the runs with processing threads enabled since one of them was failing. I did not investigate further.